src/components/dify-chatbot.module.css

```
@@ -0,0 +1,125 @@
+  }
+}
+
+:global(#dify-chatbot-bubble-button.loading),
+:global(#dify-chatbot-bubble-button[data-loading='true']) {
+  animation: pulse-glow 1.5s ease-in-out infinite !important;
+}
+
+@keyframes pulse-glow {
+  0%,
+  100% {
+    box-shadow: 0 0 20px 5px rgba(238, 120, 0, 0.6) !important;
```

Stylelint (17.3.0)

[error] Unexpected !important (keyframe-declaration-no-important)

(keyframe-declaration-no-important)